sycl/doc/extensions/supported/sycl_ext_intel_esimd/sycl_ext_intel_esimd.md

```diff
@@ -1072,3 +1100,35 @@ int main(void) {
 ```
 more examples can be found in the
 [ESIMD test suite](https://github.com/intel/llvm/tree/sycl/sycl/test-e2e/ESIMD) on github.
+
+## Implementation restrictions
+
+This section contains lists of the main restrictions that apply when using the ESIMD
+extension.
+> **Note**: Some restrictions are not enforced by the compiler, which may lead to
+> undefined program behavior if violated.
+
+### Features not supported with the ESIMD extension:
+- The [C and C++ Standard libraries support](../supported/C-CXX-StandardLibrary.rst)
+- The [Device library extensions](../../../design/DeviceLibExtensions.rst)
+
+### Unsupported standard SYCL APIs:
+
+Current ESIMD implementation does not support using certain standard SYCL features
+inside explicit SIMD kernels and functions. Most of them will be eventually
+dropped. What's not supported today:
```

Comment:
`Most of them will be eventually dropped`. What does this refer to? SYCL features below will be dropped?

Reply:
That sentence was wrong. Probably, the context before it was changed and the sentence remained as it was.
I fixed it in d6b2ac049189639af1f1a13eceaeb2a38b1235e4